docs/dev/multipath.md

```diff
@@ -0,0 +1,87 @@
+    - Installing the scsi-target-utils package: `dnf -y install scsi-target-utils`
+    -  Start the firewalld service: `sudo systemctl start firewalld`
+    - Add the iSCSI target in the firewall configuration: `firewall-cmd --zone=FedoraServer --add-service=iscsi-target --permanent`
+    - Reload the firewall configuration to apply the changes: `firewall-cmd --reload`
+    - Starts the iSCSI target daemon: `sudo systemctl start tgtd`
+    - Enable the tgtd service to start automatically at boot: `sudo systemctl enable tgtd`
+    - Create a target device: `tgtadm --lld iscsi --mode target --op new --tid=1 --targetname iqn.2009-02.com.example:assisted-installer`
+    - Add LUN to the target device: `tgtadm --lld iscsi --mode logicalunit --op new --tid 1 --lun 1 -b /dev/vdb`
+    - Configured the target to accept ALL initiators: `tgtadm --lld iscsi --mode target --op bind --tid 1 -I ALL`
+    - Show the current configuration and status of the iSCSI targets: `sudo tgtadm --mode target --op show`
+
+### STEP-3 - Create SNO VM (the initiator)
+
+- Download the ISO with SSH enabled from the Assisted Installer's UI.
+- In your VMM, create a new VM, naming it "SNO-SAN". Select the ISO you downloaded and allocate 22GB of memory, 12 CPUs, and 120GB of storage.
+- create a new record for an iSCSI target in the initiator's configuration: `iscsiadm --mode node --targetname <the name of the iSCSI target you created in SAN> --portal <SAN 1st ip>:3260 -o new`
+-  logs in to the iSCSI target, establishing a connection between the initiator and the target: `iscsiadm --mode node --targetname <the name of th iSCSI target you created in SAN> --portal <SAN 1st ip>:3260 --login`
+- Run `lsblk` to verify if an additional disk has been added. You can verify that the disk you see is the iSCSI target by using `dmesg`.
+
+### STEP-4 - 2nd Network as 2nd path to reach the iSCSI target
+
+1. In SAN VM - "show virtual hardware details" -> Add Hardware -> Network -> select the isolated network you created in STEP-1 -> copy the MAC address
+2. In SNO-SAN VM - "show virtual hardware details" -> Add Hardware -> Network -> select the isolated network you created in STEP-1 -> copy the MAC address
+3. In your beaker machine - 
+   - `sudo virsh net-edit SAN-network`
+   - In the dhcp section add the following:
+   
+   `<host mac='<SAN mac from point (1)>' name='SAN' ip='192.168.100.125'/>`
+    `<host mac='<SNO-SAN mac from point (2)>' name='SNO-SAN' ip='192.168.100.126'/>`
+
+    - `sudo virsh net-destroy SAN-network`
+    - `sudo virsh net-start SAN-network`
+    - Check that the changes updated: `sudo virsh net-dumpxml SAN-network`
+4. Click "finish" in points 1-2.
+5. In SNO-SAN VM -
+    - `iscsiadm --mode node --targetname <iSCSI target you created in SAN> --portal 192.168.100.125:3260 -o new`
+    - `iscsiadm --mode node --targetname<iSCSI target you created in SAN> --portal 192.168.100.125:3260 --login`
```

Comment:
```suggestion
    ```bash
    iscsiadm --mode node --targetname <iSCSI target you created in SAN> --portal 192.168.100.125:3260 -o new
    iscsiadm --mode node --targetname<iSCSI target you created in SAN> --portal 192.168.100.125:3260 --login
    ```
```